.github/workflows/swiftlint.yml

```diff
@@ -0,0 +1,21 @@
+name: SwiftLint
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/swiftlint.yml'
+      - '.swiftlint.yml'
+      - '**/*.swift'
+
+jobs:
+  SwiftLint:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: GitHub Action for SwiftLint
+      if: github.event.pull_request.base.repo.id == github.event.pull_request.head.repo.id
+      uses: norio-nomura/action-swiftlint@2.2.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why can the jazzy action use `GITHUB_TOKEN` without this?

Reply:
I forgot to add them to jazzy.